### PR TITLE
Book page api

### PR DESCRIPTION
**Pull request by lakshmikiran14**
@TulasiJagan49  Could you please check the api for book page?

### CODE REVIEW

File: Project 1/static/script.js [hunk 1]
```diff
@@ -0,0 +1,97 @@
+function book_html(id) {
```
Comment (TulasiJagan49) [inline Project 1/static/script.js:1]: @lakshmikiran14 Where are you using these functions defined in script.js?

Comment (TulasiJagan49) [reply Project 1/static/script.js:1]: @lakshmikiran14 I want to know in which html file you are using this script.js?

Comment (lakshmikiran14) [reply Project 1/static/script.js:1]: My task was to implement this page and this function is getting called in search api. Whenever the book is selected in the search page, this function will be called

Comment (lakshmikiran14) [reply Project 1/static/script.js:1]: As it is a single page app, i have to link it in search. I'll do it @TulasiJagan49

Comment (TulasiJagan49) [reply Project 1/static/script.js:1]: @lakshmikiran14 Okie

**Review by TulasiJagan49 — APPROVED** [review]
@lakshmikiran14 You can merge it into master branch after completing the peer reviews.

**Review by kamujusrujan — APPROVED** [review]

**Review by sourabh-swargam — APPROVED** [review]

**Pull request merged**